### PR TITLE
Add 52 manually curated SNOMED mappings to Mondo - Take 2

### PR DESCRIPTION
**Pull request by twhetzel**
I talked with Blake to sort out the proxy merge issues, but the ORCIDs were also incorrect. These issues were all updated in the gSheet so I ran the ROBOT template against `mondo-edit.obo` in a new branch, `issue_add-snomed-mappings`, to fix all issues in bulk.

The original PR is at: https://github.com/monarch-initiative/mondo/pull/7817

### CODE REVIEW

**Comment by twhetzel** [comment]
To fix the proxy merges, I discussed with Nicole and she suggested to remove the `MONDO:equivalentTo` on the xrefs that were no longer the preferred xref values based on the data used to create these new xrefs.

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -205423,7 +205434,8 @@ xref: OMIM:250100 {source="MONDO:equivalentTo", source="GARD:0003230"}
 xref: Orphanet:309263 {source="MONDO:equivalentTo"}
 xref: Orphanet:512 {source="OMIM:250100", source="GARD:0003230"}
 xref: Orphanet:751 {source="OMIM:250100"}
-xref: SCTID:238031009 {source="MONDO:equivalentTo"}
+xref: SCTID:238031009 {source="https://orcid.org/0009-0002-1597-2198"}
+xref: SCTID:44359008 {source="https://orcid.org/0009-0002-1597-2198", source="MONDO:equivalentTo"}
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:205438]: `SCTID:44359008` should be the `MONDO:preferredExternal`.

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -263928,7 +263946,8 @@ xref: NCIT:C122782 {source="MONDO:equivalentTo"}
 xref: OMIM:607015 {source="Orphanet:93476", source="MONDO:equivalentTo", source="Orphanet:93476/e"}
 xref: Orphanet:579 {source="OMIM:607015"}
 xref: Orphanet:93476 {source="MONDO:equivalentTo", source="OMIM:607015"}
-xref: SCTID:73123008 {source="MONDO:equivalentTo"}
+xref: SCTID:26745009 {source="https://orcid.org/0009-0002-1597-2198", source="MONDO:equivalentTo"}
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:263949]: `SCTID:26745009` should be the `MONDO:preferredExternal`

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -263928,7 +263946,8 @@ xref: NCIT:C122782 {source="MONDO:equivalentTo"}
...
+xref: SCTID:73123008 {source="https://orcid.org/0009-0002-1597-2198"}
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:263950]: Change back to `source="MONDO:equivalentTo"`

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -205423,7 +205434,8 @@ xref: OMIM:250100 {source="MONDO:equivalentTo", source="GARD:0003230"}
 xref: Orphanet:309263 {source="MONDO:equivalentTo"}
 xref: Orphanet:512 {source="OMIM:250100", source="GARD:0003230"}
 xref: Orphanet:751 {source="OMIM:250100"}
-xref: SCTID:238031009 {source="MONDO:equivalentTo"}
+xref: SCTID:238031009 {source="https://orcid.org/0009-0002-1597-2198"}
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:205437]: Change back to `source="MONDO:equivalentTo"`

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:263949]: fixed

File: src/ontology/mondo-edit.obo (hunk 4 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:205437]: fixed

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:205438]: fixed

File: src/ontology/mondo-edit.obo (hunk 4 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:205437]: Actually `source="MONDO:equivalentTo"` needs to be removed to fix the proxy merge issue.

File: src/ontology/mondo-edit.obo (hunk 3 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:263950]: this needs to be reverted to fix the proxy merge issue and add curator Orcid, https://orcid.org/0009-0002-1597-2198

**Comment by twhetzel** [comment]
I need information from the curator on two remaining xrefs in order to know which to add `MONDO:preferredExternal`

File: src/ontology/mondo-edit.obo [hunk 5]
```diff
@@ -206704,7 +206715,8 @@ xref: OMIM:250100 {source="MONDO:equivalentTo", source="GARD:0003230"}
...
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:206718]: Sabrina says it's fine to remove this xref.

File: src/ontology/mondo-edit.obo [hunk 6]
```diff
@@ -206704,7 +206715,8 @@ xref: OMIM:250100 {source="MONDO:equivalentTo", source="GARD:0003230"}
...
+xref: SCTID:44359008 {source="https://orcid.org/0009-0002-1597-2198", source="MONDO:equivalentTo", source="MONDO:preferredExternal"}
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:206719]: remove this `source="MONDO:preferredExternal"`

File: src/ontology/mondo-edit.obo [hunk 7]
```diff
@@ -265491,7 +265509,8 @@ xref: NCIT:C122782 {source="MONDO:equivalentTo"}
 xref: OMIM:607015 {source="Orphanet:93476", source="MONDO:equivalentTo", source="Orphanet:93476/e"}
 xref: Orphanet:579 {source="OMIM:607015"}
 xref: Orphanet:93476 {source="MONDO:equivalentTo", source="OMIM:607015"}
-xref: SCTID:73123008 {source="MONDO:equivalentTo"}
+xref: SCTID:26745009 {source="https://orcid.org/0009-0002-1597-2198", source="MONDO:equivalentTo", source="MONDO:preferredExternal"}
+xref: SCTID:73123008 {source="https://orcid.org/0009-0002-1597-2198"}
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:265513]: Remove this xref per Sabrina

File: src/ontology/mondo-edit.obo [hunk 8]
```diff
@@ -265491,7 +265509,8 @@ xref: NCIT:C122782 {source="MONDO:equivalentTo"}
 xref: OMIM:607015 {source="Orphanet:93476", source="MONDO:equivalentTo", source="Orphanet:93476/e"}
 xref: Orphanet:579 {source="OMIM:607015"}
 xref: Orphanet:93476 {source="MONDO:equivalentTo", source="OMIM:607015"}
-xref: SCTID:73123008 {source="MONDO:equivalentTo"}
+xref: SCTID:26745009 {source="https://orcid.org/0009-0002-1597-2198", source="MONDO:equivalentTo", source="MONDO:preferredExternal"}
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:265512]: remove `source="MONDO:preferredExternal"`

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@twhetzel There are many instances where there are 2 SCIT MONDO:equivalentTo. Even though there is no QC checks that will fail, we should add MONDO:preferredExternal to one of them, especially since we can see in this review that there are 2 SCIT MONDO:equivalentTo for the same term. 

Also, there a few duplicated lines which should be solved with running a normalization step.

File: src/ontology/mondo-edit.obo [hunk 9]
```diff
@@ -31904,6 +31905,8 @@ subset: rare
 xref: DOID:14253 {source="MONDO:equivalentTo"}
 xref: ICD9:362.71 {source="DOID:14253"}
 xref: MEDGEN:1843490 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
+xref: SCTID:15059000 {source="https://orcid.org/0009-0002-1597-2198", source="MONDO:equivalentTo"}
+xref: SCTID:192788009 {source="https://orcid.org/0009-0002-1597-2198", source="MONDO:equivalentTo"}
 xref: UMLS:C0154861 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:1843490"}
```
[the remark this answers is not in the document]
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:31910]: You said Monday to only add "MONDO:preferredExternal" if known.

[the remark this answers is not in the document]
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:31910]: I said that we should have a "MONDO:preferredExternal", but we don't always know which one it is.
You/We, curators, need to look at these entries and decide which one, in our opinion, should be the preferred one.

[the remark this answers is not in the document]
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:31910]: Fixed.

File: src/ontology/mondo-edit.obo [hunk 10]
```diff
@@ -44489,6 +44492,7 @@ xref: MESH:D008052 {source="DOID:3146"}
 xref: NCIT:C97092 {source="MONDO:equivalentTo"}
 xref: Orphanet:309005 {source="MONDO:equivalentTo"}
 xref: SCTID:267431006 {source="MONDO:equivalentTo"}
+xref: SCTID:402788005 {source="https://orcid.org/0009-0002-1597-2198", source="MONDO:equivalentTo"}
 xref: UMLS:C0154251 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:57587"}
```
[the remark this answers is not in the document]
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:44496]: fixed

File: src/ontology/mondo-edit.obo [hunk 11]
```diff
@@ -45291,6 +45295,7 @@ xref: NANDO:2100165 {source="MONDO:NANDO", source="https://orcid.org/0000-0003-0
 xref: NCIT:C61250 {source="DOID:3211", source="MONDO:equivalentTo"}
 xref: Orphanet:68366 {source="MONDO:equivalentTo"}
 xref: SCTID:23585005 {source="DOID:3211", source="MONDO:equivalentTo"}
+xref: SCTID:28821000119102 {source="https://orcid.org/0009-0002-1597-2198", source="MONDO:equivalentTo"}
```
[the remark this answers is not in the document]
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:45298]: Checking with data submitter since SCTID:28821000119102 is not a valid SNOMED ID.

**Comment by blakebyer** [comment]
| MONDO         | Match in MONDO (Existing xref) | Add to MONDO (PreferredExternal) |
|---------------|-----------------------|----------------------------------|
| MONDO:0001083 | SCTID:40488004        | SCTID:236466005                  |
| MONDO:0002525 | SCTID:267431006       | SCTID:402788005                  |
| MONDO:0002561 | SCTID:23585005        | SCTID:28821000119102             |
| MONDO:0009499 | SCTID:189979005       | SCTID:192782005                  |
| MONDO:0009590 | SCTID:68390005        | SCTID:297278001                  |
| MONDO:0009590 | SCTID:68390005        | SCTID:1003375005                 |
| MONDO:0009591 | SCTID:238031009       | SCTID:44359008                   |
| MONDO:0009653 | SCTID:111384001       | SCTID:725296006                  |
| MONDO:0009661 | SCTID:69463008        | SCTID:52677002                   |
| MONDO:0009757 | SCTID:18927009        | SCTID:67855008                   |
| MONDO:0010100 | SCTID:111385000       | SCTID:49562005                   |
| MONDO:0011759 | SCTID:73123008        | SCTID:26745009                   |
| MONDO:0018149 | SCTID:124465002       | SCTID:238025006                  |
| MONDO:0018868 | SCTID:66521008        | SCTID:238031009                  |
| MONDO:0018868 | SCTID:66521008        | SCTID:396338004                  |
| MONDO:0019148 | SCTID:82500001        | SCTID:238074007                  |
| MONDO:0019346 | SCTID:34960006        | SCTID:723675006                  |
| MONDO:0800088 | SCTID:254069004       | SCTID:279081001                  |

**Comment by blakebyer** [comment]
Hi all, above is the table for where there are two equivalent SCTID xrefs. The left column is the MONDO ID, the middle column is the existing xref, and the right column is the preferred xref. Let me know if there are any other QC issues I can help with.

**Comment by blakebyer** [comment]
There was a previous problem with the following. This SCTID is in the SNOMED CT browser but not in bioportal.
| MONDO | Match in MONDO (Existing xref) | Add to MONDO (PreferredExternal)
| -----------| ------------------------------------ | -------------------------------------- | 
| MONDO:0002561 | SCTID:23585005 | SCTID:28821000119102 | 

If this continues to cause problems, I recommend just using the existing xref.

**Comment by blakebyer** [comment]
> MONDO	Match in MONDO (Existing xref)	Add to MONDO (PreferredExternal)
> MONDO:0001083	SCTID:40488004	SCTID:236466005
> MONDO:0002525	SCTID:267431006	SCTID:402788005
> MONDO:0002561	SCTID:23585005	SCTID:28821000119102
> MONDO:0009499	SCTID:189979005	SCTID:192782005
> **MONDO:0009590	SCTID:68390005	SCTID:297278001
> MONDO:0009590	SCTID:68390005	SCTID:1003375005**
> MONDO:0009591	SCTID:238031009	SCTID:44359008
> MONDO:0009653	SCTID:111384001	SCTID:725296006
> MONDO:0009661	SCTID:69463008	SCTID:52677002
> MONDO:0009757	SCTID:18927009	SCTID:67855008
> MONDO:0010100	SCTID:111385000	SCTID:49562005
> MONDO:0011759	SCTID:73123008	SCTID:26745009
> MONDO:0018149	SCTID:124465002	SCTID:238025006
> MONDO:0018868	SCTID:66521008	SCTID:238031009
> MONDO:0018868	SCTID:66521008	SCTID:396338004
> MONDO:0019148	SCTID:82500001	SCTID:238074007
> MONDO:0019346	SCTID:34960006	SCTID:723675006
> MONDO:0800088	SCTID:254069004	SCTID:279081001

Ignore these bolded lines, they were removed from the ROBOT template due to low confidence and 2 synonyms for one term.

**Comment by twhetzel** [comment]
@blakebyer which should be the prefExt here for `MONDO:0018868`:

MONDO:0018868 | SCTID:66521008 | SCTID:238031009
-- | -- | --
MONDO:0018868 | SCTID:66521008 | SCTID:396338004

**Comment by blakebyer** [comment]
@twhetzel MONDO:0018869 | SCTID:396338004

File: src/ontology/mondo-edit.obo (hunk 11 above)
[the remark this answers is not in the document]
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:45298]: fixed

File: src/ontology/mondo-edit.obo [hunk 12]
```diff
@@ -204468,6 +204476,7 @@ xref: OMIM:245200 {source="DOID:10587", source="Orphanet:487/e", source="MONDO:e
 xref: Orphanet:487 {source="MONDO:equivalentTo", source="OMIM:245200"}
 xref: SCTID:189979005 {source="DOID:10587", source="MONDO:equivalentTo"}
 xref: SCTID:192782005 {source="DOID:10587"}
+xref: SCTID:192782005 {source="https://orcid.org/0009-0002-1597-2198", source="MONDO:equivalentTo"}
```
[the remark this answers is not in the document]
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:204479]: fixed

File: src/ontology/mondo-edit.obo [hunk 13]
```diff
@@ -206976,6 +206985,8 @@ xref: NANDO:1200082 {source="MONDO:NANDO", source="https://orcid.org/0000-0003-0
 xref: NANDO:2201205 {source="MONDO:NANDO", source="https://orcid.org/0000-0003-0011-764X", source="https://orcid.org/0000-0002-0170-9172"}
 xref: OMIM:249900 {source="MONDO:equivalentTo"}
 xref: Orphanet:512 {source="OMIM:249900"}
+xref: SCTID:1003375005 {source="https://orcid.org/0009-0002-1597-2198", source="MONDO:equivalentTo"}
+xref: SCTID:297278001 {source="https://orcid.org/0009-0002-1597-2198", source="MONDO:equivalentTo"}
```
[the remark this answers is not in the document]
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:206989]: @blakebyer we missed this one

File: src/ontology/mondo-edit.obo [hunk 14]
```diff
@@ -208763,6 +208774,7 @@ xref: NORD:1460 {source="MONDO:NORD"}
 xref: OMIM:252650 {source="Orphanet:578", source="MONDO:equivalentTo", source="Orphanet:578/e", source="GARD:0000094"}
 xref: Orphanet:578 {source="MONDO:equivalentTo", source="OMIM:252650", source="GARD:0000094"}
 xref: SCTID:111384001 {source="MONDO:equivalentTo"}
+xref: SCTID:725296006 {source="https://orcid.org/0009-0002-1597-2198", source="MONDO:equivalentTo"}
```
[the remark this answers is not in the document]
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:208777]: fixed

File: src/ontology/mondo-edit.obo [hunk 15]
```diff
@@ -209120,6 +209132,7 @@ xref: OMIM:253200 {source="DOID:12800", source="Orphanet:583/e", source="MONDO:e
 xref: Orphanet:583 {source="MONDO:equivalentTo", source="OMIM:253200"}
 xref: SCTID:190936000 {source="DOID:12800"}
 xref: SCTID:52677002 {source="DOID:12800"}
+xref: SCTID:52677002 {source="https://orcid.org/0009-0002-1597-2198", source="MONDO:equivalentTo"}
 xref: SCTID:69463008 {source="DOID:12800", source="MONDO:equivalentTo"}
```
[the remark this answers is not in the document]
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:209136]: fixed

File: src/ontology/mondo-edit.obo [hunk 16]
```diff
@@ -212320,6 +212334,7 @@ xref: NCIT:C126864 {source="MONDO:equivalentTo"}
 xref: OMIM:257220 {source="DOID:0070113", source="MONDO:equivalentTo"}
 xref: Orphanet:646 {source="OMIM:257220"}
 xref: SCTID:18927009 {source="MONDO:equivalentTo"}
+xref: SCTID:67855008 {source="https://orcid.org/0009-0002-1597-2198", source="MONDO:equivalentTo"}
```
[the remark this answers is not in the document]
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:212337]: fixed

File: src/ontology/mondo-edit.obo [hunk 17]
```diff
@@ -222055,6 +222070,7 @@ xref: NORD:1761 {source="MONDO:NORD"}
 xref: OMIM:272800 {source="DOID:3320", source="Orphanet:845", source="MONDO:equivalentTo", source="Orphanet:845/e"}
 xref: Orphanet:845 {source="OMIM:272800", source="MONDO:equivalentTo"}
 xref: SCTID:111385000 {source="DOID:3320", source="MONDO:equivalentTo"}
+xref: SCTID:49562005 {source="https://orcid.org/0009-0002-1597-2198", source="MONDO:equivalentTo"}
```
[the remark this answers is not in the document]
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:222073]: fixed

File: src/ontology/mondo-edit.obo [hunk 18]
```diff
@@ -411649,6 +411685,7 @@ xref: NANDO:2200558 {source="MONDO:NANDO", source="https://orcid.org/0000-0003-0
 xref: NCIT:C84739 {source="DOID:3322", source="MONDO:equivalentTo"}
 xref: Orphanet:354 {source="MONDO:equivalentTo"}
 xref: SCTID:124465002 {source="MONDO:equivalentTo"}
+xref: SCTID:238025006 {source="https://orcid.org/0009-0002-1597-2198", source="MONDO:equivalentTo"}
```
[the remark this answers is not in the document]
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:411688]: fixed

File: src/ontology/mondo-edit.obo (hunk 13 above)
[the remark this answers is not in the document]
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:206989]: preferred Ext is SCTID:44359008

**Comment by blakebyer** [comment]
MONDO:0009591 | SCTID:238031009 |  SCTID:44359008

**Comment by blakebyer** [comment]
Okay the preferred xref for MONDO:0009590 is SCTID:1003375005

File: src/ontology/mondo-edit.obo (hunk 13 above)
[the remark this answers is not in the document]
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:206989]: [From Blake](https://github.com/monarch-initiative/mondo/pull/7950#issuecomment-2265931941): Okay the preferred xref for MONDO:0009590 is SCTID:1003375005

[the remark this answers is not in the document]
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:206989]: fixed

File: src/ontology/mondo-edit.obo [hunk 19]
```diff
@@ -433864,6 +433904,7 @@ xref: Orphanet:75233 {source="MONDO:equivalentTo"}
 xref: SCTID:190795007 {source="DOID:14497"}
 xref: SCTID:190797004 {source="DOID:14497"}
 xref: SCTID:238074007 {source="MONDO:possiblyEquivalent", source="DOID:14497"}
```
[the remark this answers is not in the document]
Comment (blakebyer) [reply src/ontology/mondo-edit.obo]: Should be source="MONDO:equivalentTo"

**Review by sabrinatoro — APPROVED** [review]
Everything looks good. 
Thank you @twhetzel and @blakebyer

**Pull request merged**